time stamp
uint16[] id_list

bool[] input_valtage
bool[] motor_hall_sensor
bool[] overheating
bool[] motor_encoder
bool[] electronical_shock
bool[] overload
